uwblueprint/childrens-aid-society
# Create Intake Goals Table

## Notion ticket link
[Create intake<>goals association table](https://www.notion.so/uwblueprintexecs/Create-intake-goals-association-table-b848ad6ed1394c2eaf0977278c217ac9)



## Implementation description
* Create `intake_goals` table for many to many mapping between intake and goals
* Created db migration file



## Steps to test
1. Verify migration applies with no errors by running `docker exec -it cas_py_backend /bin/bash -c "flask db upgrade"` and then `docker exec -it cas_py_backend /bin/bash -c "flask db downgrade"`
2. Run `docker exec -it cas_py_backend /bin/bash -c "flask db upgrade"` again. Run `docker exec -it cas_py_backend /bin/bash -c "flask db current -v"` and verify that the current message is "Create intake goals association table"
3. Run `docker exec -it cas_db /bin/bash -c "psql -U postgres -d cas"`
4. `SELECT * FROM intakes_goals;` Then verify that the following table appears.
<img width="327" alt="image" src="https://user-images.githubusercontent.com/55464415/179413228-8385111b-cded-447f-bdf2-c09be31e81ae.png">


## What should reviewers focus on?
* Migration file has correct types and nullable values


## Checklist
